.github/workflows/release.yml

```diff
@@ -0,0 +1,164 @@
+name: Release new version of Optable SDK and demos
+on:
+  push:
+    tags:
+      - "v.*"
+
+jobs:
+  tests-prettier:
+    uses: ./.github/workflows/reusable-lint-test.yml
+
+  build:
+    needs: [tests-prettier]
+    uses: ./.github/workflows/reusable-build.yml
+
+  deploy-sdk-to-npm:
+    needs: [build]
+    runs-on: ubuntu-latest
+    env:
+      gcp-project-id: 118585658141
+      service-account: gh-ci-optable-web-sdk
```

Comment:
Since you are using this in multiple places, maybe we can make it outside the scope of the jobs so you declare it only once, and it will be available to all jobs.